front/src/modules/pathfinding/components/Itinerary/ModalSuggestedVias.tsx

```diff
@@ -0,0 +1,126 @@
+import { formatUicToCi } from 'utils/strings';
+
+type ModalSuggestedViasProps = {
+  suggestedOps: SuggestedOP[];
+};
+
+const ModalSuggestedVias = ({ suggestedOps }: ModalSuggestedViasProps) => {
+  const { updatePathSteps, upsertViaFromSuggestedOP, clearViasV2 } = useOsrdConfActions();
+  const { getViasV2, getDestinationV2, getPathSteps } = useOsrdConfSelectors();
+  const dispatch = useAppDispatch();
+  const vias = useSelector(getViasV2());
+  const destination = useSelector(getDestinationV2);
+  const pathSteps = useSelector(getPathSteps);
+  const { t } = useTranslation('operationalStudies/manageTrainSchedule');
+
+  const isOriginOrDestination = useCallback(
+    (op: SuggestedOP) =>
+      op.positionOnPath === 0 || op.positionOnPath === destination?.positionOnPath,
+    [destination]
+  );
+
+  const removeViaFromPath = (op: SuggestedOP) => {
+    const updatedPathSteps = [...pathSteps];
+    dispatch(
+      updatePathSteps(
+        compact(updatedPathSteps).filter(
+          (step) =>
+            ('uic' in step && step.uic !== op.uic) || ('track' in step && step.track !== op.track)
+        )
+      )
+    );
+  };
+
+  const formatOP = (op: SuggestedOP, idx: number, idxTrueVia: number) => {
+    const isInVias = isVia(vias, op);
+    return (
+      <div
+        key={`suggested-via-modal-${op.opId}-${idx}`}
+        className={cx('d-flex align-items-center p-1', { 'suggested-via-clickable': !isInVias })}
+        title={op.name}
+      >
+        {isInVias && <small className="pr-2">{idxTrueVia}</small>}
+        <i className={`${!isInVias ? 'text-muted' : 'text-info'} icons-itinerary-bullet mr-2`} />
+        <span className="suggested-via-name">{op.name || ''}</span>&nbsp;
+        {op.ch && <span>{op.ch}</span>}
+        {op.uic && <small className="text-muted ml-3">{formatUicToCi(op.uic)}</small>}
+        <div className="ml-auto">
+          {op.positionOnPath && (
+            <small className="mr-2">{`KM ${(Math.round(op.positionOnPath) / 1000000).toFixed(3)}`}</small>
+          )}
+          {!isInVias ? (
+            <button
+              className="btn btn-sm btn-only-icon"
+              type="button"
+              aria-label={t('addVia')}
+              title={t('addVia')}
+              onClick={() => dispatch(upsertViaFromSuggestedOP(op))}
+            >
+              <Plus />
+            </button>
+          ) : (
+            <button
+              className="btn btn-sm btn-only-icon bg-dark"
+              type="button"
+              aria-label={t('removeVia')}
+              title={t('removeVia')}
+              onClick={() => removeViaFromPath(op)}
+            >
+              <Dash />
+            </button>
+          )}
+        </div>
+      </div>
+    );
+  };
+
+  let idxTrueVia = 0;
+  return (
+    <div className="manage-vias-modal">
+      <ModalHeaderSNCF withCloseButton>
+        <h1>{`${t('manageVias', { count: vias.length, viasCount: vias.length })}`}</h1>
```

Comment:
You can directly use count
```suggestion
        <h1>{`${t('manageVias', { count: vias.length })}`}</h1>
```

Reply:
Oh yes, my bad